src/ser.rs

```diff
@@ -214,6 +215,7 @@ impl ser::SerializeStruct for ObjectSerializer<'_> {
 #[derive(Default)]
 pub struct Serializer {
     serialize_maps_as_objects: bool,
+    serialize_64_bit_numbers_as_big_int:bool
```

Comment:
(here and around) please make sure to run `cargo fmt` in the end, or, better yet, you might want to configure format-on-save in your editor.